spec/Procedures.tex

```diff
@@ -537,7 +537,8 @@ \subsubsection{The Const Intent}
 
 \begin{craychapel}
 The current implementation uses the following mapping:
-
+% TODO: does this really need to be about the "current implementation"?
+% Couldn't it just be about "the current language"?
```

Comment:
I agree that we should make it a language choice, not an implementation choice.  Assuming there's consensus on that, you can also remove the "Open issue" section just above this section.